xmppserver/src/test/java/org/jivesoftware/openfire/streammanagement/StreamManagerTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015-2023 Ignite Realtime Foundation. All rights reserved.
+ * Copyright (C) 2023 Ignite Realtime Foundation. All rights reserved.
```

Comment:
I don't understand this one @guusdk.
You added this test file in 2023, but marked it as 2015-2023.
Reckon this was a copy/paste error, or was there some intent?

Reply:
There unlikely was intent. It is likely a copy/paste thing.